public/index.html

```diff
@@ -9,6 +9,13 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <style>
+      @import url('http://fonts.cdnfonts.com/css/gill-sans');
+    </style>
+
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+      </style>
```

Comment:
- [ ] _[OPTIONAL]_ Please move these imports to the index.css file instead of adding two style tags.